README.md

```diff
@@ -22,79 +30,90 @@ Also, add the applinks:<APPSITE HOST> entry to the Associated Domains Capability
+  "ssl-required": "string",
+  "resource": "<resource_name>",
+  "credentials": {
+    "secret": "<secret_uuid>"
+  },
+  "confidential-port": "string",
+}
 ```
 
-### Login
-```js
-
-const config = {
-  url: 'https://<KEYCLOAK_HOST>/auth',
-  realm: '<REALM NAME>',
-  resource: '<RESOURCE NAME>',
-  redirectUri: 'https://<REDIRECT HOST>/success.html',
-  appsiteUri: 'https://<APPSITE HOST>/app.html',
-  kcIdpHint: 'facebook', // *optional*
-};
-
-Login.startLoginProcess(config).then(tokens => {
-  console.log(tokens);
-});
-
-// Prints:
-//
-// { access_token: '...', refresh_token: '...', id_token: '...', ...}
+Resolver arguments:
+ - _response_: a JSON object containing two fields:
+    - *tokens*: a JSON containing all the tokens returned by KeyCloak. If you used'info' as *scope* the JSON will be as shown below.
+    - *deepLinkUrl*: The redirectUrl with some KeyCloak query params added at the end.
+
+```json
+response.tokens = {
+    "access_token": "string",
+    "expires_in": "number",
+    "refresh_expires_in": "number",
+    "refresh_token": "string",
+    "token_type": "string",
+    "not-before-policy": "number",
+    "session_state": "string",
+    "scope": "string",
+}
 ```
 
-Logging in by the startLoginProcess function will save it in the AsyncStorage, whereas after its been successful, getTokens will get the most recent tokens that are saved and you can then use it to authenticate against a backend.
+#### Manually handling the tokens
 
-### Refreshing the token
 ```js
-const refreshedTokens = await Login.refreshToken();
-console.log(refreshTokens);
-// Prints:
-//
-// { access_token: '...', refresh_token: '...', id_token: '...', ...}
-// OR
-// undefined
+import KeyCloak, { TokenStorage } from 'react-native-keycloak-plugin'
 ```
 
+Logging in by the login function will save the tokens information into the AsyncStorage. Through the TokenStoage object, the plugin export three methods that can be used to get, save and clear the tokens. This method are needed if you want to directly access and manage the tokens from the AsyncStorge.
 
-
-### Retrieving logged in user info
+### KeyCloak.retrieveUserInfo
 ```js
-const loggedInUser = await Login.retrieveUserInfo();
-console.log(loggedInUser);
-
-// Prints:
-//
-// { sub: '...',name: '... ',preferred_username: '...',given_name: '...' }
-
-// OR
-// undefined
+KeyCloak.retrieveUserInfo(conf)
+  .then((userInfo) => /* Your resolve */ );
+  .catch((error) => /* Your reject*/ )
 ```
+Passing a configuration JSON object, makes available into the resolve funcion the JSON that describe the user inside KeyCloak.
 
+### KeyCloak.refreshToken
+```js
+KeyCloak.refreshToken(conf)
+  .then((response) => /* Your resolve */ );
+  .catch((error) => /* Your reject*/ )
+```
+Passing a configuration JSON object, makes available into the resolve funcion the JSON contains the refreshed tokens. This information are also saved into the AsyncStorage described above.
```

Comment:
Typo: "function" not "funcion"
Grammar: "The JSON containing the refreshed tokens", not "contains"
Tip: in the last line it's better to say "... AsyncStorage, AS described above"